commands/update.go

```diff
@@ -0,0 +1,193 @@
+	RunE: func(cmd *cobra.Command, args []string) error {
+		return updateRepos()
+	},
+	Args: func(cmd *cobra.Command, args []string) error {
+		if len(sourceFile) > 0 && len(sourceRepo) > 0 {
+			return fmt.Errorf("both --source-file and --source-repo cannot be set")
+		}
+
+		var authenticated bool
+		if err := storage.Read(func(table *storage.Table) {
+			authenticated = len(table.GithubAccessToken) > 0
+		}); err != nil {
+			return err
+		}
+
+		if !authenticated {
+			return fmt.Errorf("not authenticated")
+		}
+
+		return nil
+	},
+}
+
+func updateRepos() error {
+	var authToken string
+	var updatedRepos []string
+	var transferredRepos []storage.Repo
+
+	if err := storage.Read(func(table *storage.Table) {
+		authToken = table.GithubAccessToken
+		updatedRepos = table.UpdatedRepositories
+		transferredRepos = table.TransferredRepositories
+	}); err != nil {
+		return err
+	}
+
+	for _, repo := range transferredRepos {
+		from := repoOwner(repo.OriginalURL) + "/" + repoName(repo.OriginalURL)
+		to := repo.TargetOrg + "/" + repoName(repo.OriginalURL)
+		newURL := strings.Replace(repo.OriginalURL, from, to, -1)
+
+		if sliceutil.IsStringInSlice(newURL, updatedRepos) {
+			fmt.Println("Skip (already updated):", newURL)
+			continue
+		}
+
+		tempDir, err := pathutil.NormalizedOSTempDirPath("repo")
+		if err != nil {
+			log.Errorf("%s", err)
+			continue
+		}
+
+		if out, err := command.New("git", "clone", newURL, tempDir).RunAndReturnTrimmedCombinedOutput(); err != nil {
+			log.Errorf("%s", errors.Wrap(err, out))
+			continue
+		}
+
+		if out, err := command.New("git", "branch", "git-url-update").SetDir(tempDir).RunAndReturnTrimmedCombinedOutput(); err != nil {
+			log.Errorf("%s", errors.Wrap(err, out))
+			continue
+		}
+
+		if out, err := command.New("git", "checkout", "git-url-update").SetDir(tempDir).RunAndReturnTrimmedCombinedOutput(); err != nil {
+			log.Errorf("%s", errors.Wrap(err, out))
+			continue
+		}
+
+		if err := filepath.Walk(tempDir, func(path string, info os.FileInfo, err error) error {
+			if err != nil {
+				return err
+			}
+
+			if info.IsDir() && info.Name() == ".git" {
+				return filepath.SkipDir
+			}
+
+			if info.IsDir() {
+				return nil
+			}
+
+			c, err := ioutil.ReadFile(path)
+			if err != nil {
+				return err
+			}
+
+			c = []byte(strings.Replace(string(c), from, to, -1))
+
+			return ioutil.WriteFile(path, c, info.Mode())
+		}); err != nil {
+			log.Errorf("%s", err)
+			continue
+		}
+
+		if out, err := command.New("git", "add", ".").SetDir(tempDir).RunAndReturnTrimmedCombinedOutput(); err != nil {
+			log.Errorf("%s", errors.Wrap(err, out))
+			continue
+		}
+
+		if out, err := command.New("git", "commit", "-m", "updated git urls").SetDir(tempDir).RunAndReturnTrimmedCombinedOutput(); err != nil {
+			log.Errorf("%s", errors.Wrap(err, out))
+			continue
+		}
+
+		if out, err := command.New("git", "push", "origin", "git-url-update").SetDir(tempDir).RunAndReturnTrimmedCombinedOutput(); err != nil {
```

Comment:
The branch does not exist on the `origin` repository at this point. You will need the `--set-upstream` (`-u`) flag for a successful push.